File: FrizerskiSalon.Modal/TipTermina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FrizerskiSalon.Modal
{
    public class TipTermina
    {
        public int TipTerminaId { get; set; }
        public string Naziv { get; set; }

        public override string ToString()
        {
            return Naziv;
        }
    }
    
}

File: FrizerskiSalon.Mobile/frizerski_salon_app/lib/model/TipTermina.dart
class TipTermina{
  int tipTerminaId;
  String naziv;

  TipTermina({
    required this.tipTerminaId,
    required this.naziv,
});

  factory TipTermina.fromJson(Map<String, dynamic> json) {
    return TipTermina(
        tipTerminaId: int.parse(json["tipTerminaId"].toString()),
        naziv: json["naziv"]
    );
  }

  Map<String, dynamic> toJson() => {
    "tipTerminaId":tipTerminaId,
    "naziv":naziv,
  };
}